lib/standard/stream.nit

```diff
@@ -392,6 +392,34 @@ abstract class BufferedReader
 		return c.ascii
 	end
 
+	# Peeks up to `n` bytes in the buffer, returns an empty string on EOF
+	#
+	# The operation does not consume the buffer
+	#
+	# ~~~nitish
+	# 	var x = new FileReader("File.txt")
+	#	assert x.peek(5) == x.read(5)
+	# ~~~
```

Comment:
You don't need the indent with fences, this indent will be used verbatim.

By the way, could you give a better name to `x`? Maybe `stream`, `file`, `reader` or even `r` or `f`... :wind_chime: